ss/ss.go

```diff
@@ -18,24 +19,14 @@ import (
 )
 
 const (
-	processeNameFieldIdx  = 5
 	localAddrPortFieldIdx = 3
 	interval              = time.Millisecond * 500
 	duration              = time.Second * 5
 )
 
-var (
-	// TcpSSFilterFn is a function variable in Go that filters entries from the 'ss' command output.
-	// It takes an entry from the 'ss' command output and returns true if the entry represents a TCP port in the listening state.
-	tcpSSFilterFn = func(s string) bool {
-		return strings.Contains(s, "127.0.0") || !strings.Contains(s, "LISTEN")
-	}
-	// UdpSSFilterFn is a function variable in Go that filters entries from the 'ss' command output.
-	// It takes an entry from the 'ss' command output and returns true if the entry represents a UDP port in the listening state.
-	udpSSFilterFn = func(s string) bool {
-		return strings.Contains(s, "127.0.0") || !strings.Contains(s, "ESTAB")
-	}
-)
+var filterOutFn = func(s string) bool {
```

Comment:
This one is filtering UDP....
The right filter will be
```
var ssFilterFn = func(s string) bool {
		return strings.Contains(s, "127.0.0") || strings.Contains(s, "::1") || s == ""
}
```
Changing to this one might cause you other issues I have fixed in my PR

Reply:
fixed